tikv-client/src/test/java/com/pingcap/tikv/expression/visitor/CanBePrunedValidatorTest.java

```diff
@@ -0,0 +1,74 @@
+package com.pingcap.tikv.expression.visitor;
+
+import static com.pingcap.tikv.expression.visitor.ExpressionTypeCoercerTest.createTable;
+import static org.junit.Assert.*;
+
+import com.pingcap.tikv.expression.ColumnRef;
+import com.pingcap.tikv.expression.ComparisonBinaryExpression;
+import com.pingcap.tikv.expression.Constant;
+import com.pingcap.tikv.expression.Expression;
+import com.pingcap.tikv.expression.LogicalBinaryExpression;
+import com.pingcap.tikv.expression.visitor.CanBePrunedValidator.ContextPartExpr;
+import com.pingcap.tikv.meta.TiTableInfo;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CanBePrunedValidatorTest {
+
+  @Test
+  public void canBePrunedOrWithUnrelatedColCase() {
+    TiTableInfo tableInfo = createTable();
```

Comment:
do we need to add partition info in createTable?

Reply:
No. This is just to verify the correctness of canBePrunedValidator .